script.js

```diff
@@ -36,18 +36,23 @@ function setup() {
 	for (let i = 0; i < modes.length; i++) {
 		let mode = modes[i]
 		let button = createButton(mode.name)
+    button.attribute('name', mode.name)
 		button.size(BUTTON_WIDTH, BUTTON_HEIGHT)
 		button.style('font-size', '24px')
 		button.position(
 			(width - BUTTON_WIDTH) / 2,
       //700 was originally "height", but it messes up on smaller viewports. Not entirely sure why.
 			(700 - modes.length * (BUTTON_HEIGHT + BUTTON_GAP) * i) / 2 - BUTTON_GAP
 		)
-		button.mousePressed(() => {
-			currentMode = modes[i]
-			modeSelectButtons.forEach((b) => b.hide())
-			state = PLAY_STATE
-		})
+    button.mousePressed(() => {
+      currentMode = modes[i]
+      modeSelectButtons.forEach((b) => b.hide())
+      if (currentMode.name == 'Grade Ten+') {
```

Comment:
Why woudl the imported phrases be restricted to 'grade 10+' mode? Only?  If I'm a teacher, I want to set words for my class and play... whatever grade/mode I choose.

Reply:
I don't envision a user importing a different set of phrases for each mode.  Typically a user is only interested in one mode (or some themed variant).  The modes are there to give choices to different types of users.  For example, a Grade 1 teacher would probably only care about the grade 1 mode.  Highschool teacher probably only cares about the customizing another mode.  We have no persistence right now anyway (no plan for it) so I don't think users are going to take the time to customize more than one mode for their people to play; when things will be forgotten upon reload.

Reply:
Ok, I'll get it working for both.